jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java

```diff
@@ -104,11 +107,34 @@ private static void assertExtractionPathsUnordered(
   @Mock private AuthProperty authProperty;
   @Mock private Consumer<LogEvent> logger;
 
+  /** Configuration for {@code platform} parameter. */
+  public static class PlatformParametersTest implements PlatformConfiguration {
+    @Nullable private String os;
+    @Nullable private String architecture;
+
+    PlatformParametersTest(String architecture, String os) {
+      this.architecture = architecture;
+      this.os = os;
+    }
+
+    @Override
+    public Optional<String> getOsName() {
+      return Optional.ofNullable(os);
+    }
+
+    @Override
+    public Optional<String> getArchitectureName() {
+      return Optional.ofNullable(architecture);
+    }
+  }
+
   @Before
   public void setUp() throws IOException, InvalidImageReferenceException, InferredAuthException {
     Mockito.when(rawConfiguration.getFromAuth()).thenReturn(authProperty);
     Mockito.when(rawConfiguration.getEntrypoint()).thenReturn(Optional.empty());
     Mockito.when(rawConfiguration.getAppRoot()).thenReturn("/app");
+    Mockito.<List<? extends PlatformConfiguration>>when(rawConfiguration.getPlatforms())
```

Comment:
Probably `Mockito.when(...)` will just work.

Reply:
We need to chat in person about this implementation ,I might some help to change it.

Reply:
Ah, just realized we can't remove it. I think in our test code, we can just say `Mockito.<List<?>>`.

Reply:
That implemantain works